### ISSUE TITLE
IndexError: list assignment index out of range

### ISSUE THREAD

**Issue by diminoten** [PyCQA/pep8#388]
Got this while running pep8 tool against some functional tests:

```Python
Traceback (most recent call last):
  File "/Users/diminoten/git/project/ENV/bin/pep8", line 11, in <module>
    sys.exit(_main())
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 2107, in _main
    report = pep8style.check_files()
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 1830, in check_files
    self.input_dir(path)
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 1866, in input_dir
    runner(os.path.join(root, filename))
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 1844, in input_file
    return fchecker.check_all(expected=expected, line_offset=line_offset)
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 1608, in check_all
    self.check_logical()
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 1490, in check_logical
    for offset, text in self.run_check(check, argument_names) or ():
  File "/Users/diminoten/git/project/ENV/lib/python2.7/site-packages/pep8.py", line 573, in continued_indentation
    rel_indent[end[0] - first_row] = rel_indent[row]
IndexError: list assignment index out of range
```

Haven't investigated much further.

**Comment by sigmavirus24** [comment]
Can you provide information to reproduce this with? 
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by diminoten** [comment]
Not sure which file in my project it's choking on.

Does it just run through the directory alphabetically?

**Comment by IanLee1521** [comment]
Try running with verbose or debug output:

pep8 -v ...
pep8 -vv ...

**Comment by sigmavirus24** [comment]
Yes
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by diminoten** [comment]
Can repro it with this script:

```Python
foo = \
"""foo
bar"""
```

**Comment by diminoten** [comment]
`# noqa` fixes crash per #45 dunno if that means this can close or not

**Comment by sigmavirus24** [comment]
```
❯❯❯ cat pep8_bug388.py
foo = \
"""foo
bar"""
❯❯❯ pep8 --version
1.5.7
❯❯❯ pep8 pep8_bug388.py
pep8_bug388.py:2:1: E122 continuation line missing indentation or outdented
❯❯❯ python pep8.py --version
1.6.3a0
❯❯❯ python pep8.py pep8_bug388.py
pep8_bug388.py:2:1: E122 continuation line missing indentation or outdented
```

So I can't reproduce this. It seems like @diminoten had some kind of work-around for this as well. So I'm going to close this. If this rears its head again, though, we should re-open this and re-investigate it.
